.github/workflows/main.yml

```
@@ -0,0 +1,21 @@
+name: Collector CI
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  pull_request:
+    branches:
+      - master
+    types: [opened, reopened, synchronize, labeled]
+
+jobs:
+  build-collector:
+    uses: ./.github/workflows/collector-image.yml
+    secrets: inherit
+
+  integration-tests:
+    uses: ./.github/workflows/integration-tests.yml
+    needs: [build-collector]
```

[nit] When the requirement is a single job, we don't need to use an array
```suggestion
    needs: build-collector
```